Automattic/jetpack
# My Jetpack: Move VaultPress Backup out of hybrid concept

This one moves VaultPress Backup out of the concept of a hybrid product.

So, we will prioritize only the standalone plugin and not rely on the Jetpack module itself.

#### Changes proposed in this Pull Request:

* Make `class-backup` extends from `Product` and not `HybridProduct`.
* Manage always goes for the `VaultPress Backup` dashboard.

#### Other information:


#### Jetpack product discussion


p1HpG7-jwG-p2

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Setup a new JN site only with the Jetpack plugin
* Goes to My Jetpack page.
* VaultPress Backup should be marked as absent -  `Add VaultPressBackup` should be available.
* Goes into adding it.
* It now should be active, and manage prompts the user to the `VaultPress Backup` dashboard.
* Goes into the plugin page and deactivates it.
* Now, My Jetpack should reflect as inactive.
